app/styles/base-styles/_z-idex-scale.scss

```diff
@@ -0,0 +1,17 @@
+// Z-Index Scale
```

Comment:
It'd be great to have a page on wikiadesignsystem.com which has some short documentation